invoker/core/src/main/java/com/google/cloud/functions/invoker/runner/Invoker.java

```diff
@@ -62,6 +63,7 @@
 import org.eclipse.jetty.http.HttpStatus;
 import org.eclipse.jetty.server.Request;
 import org.eclipse.jetty.server.Server;
+import org.eclipse.jetty.server.ServerConnector;
```

Comment:
We add 2 imports, are they used somewhere?

Reply:
Nope, that was while I was working on an earlier fix. Removed.